examples/opentelemetry-tracing/Dockerfile-opentelemetry-collector

```diff
@@ -0,0 +1 @@
+FROM otel/opentelemetry-collector:latest
```

Comment:
can you add the `@sha:xyz` here as then its pinned, and will get updated by dependabot